driver/src/orderbook/streamed/state.rs

```diff
@@ -33,8 +45,31 @@ struct LastSolution {
 }
 
 impl State {
-    /// Can be used to create a `crate::models::AccountState`.
-    pub fn account_state(
+    /// Errors if State has only received a partial solution which would make the result
+    /// inconsistent.
+    /// Errors if the batch is `Future` but is not actually in the future.
+    /// Account balances that overflow a U256 are skipped.
+    pub fn orderbook(
```

Comment:
uber naming nit: I think in the follow up PR it reads slightly nicer if this was called `get_orderbook` or even `get_orderbook_for_batch`.

Reply:
I like `orderbook_for_batch`. I'm fine with leaving out the `get_` is its pretty common in Rust for getters to simply be called what is being retrieved (such as `str::chars` instead of `str::get_chars`).